impeller/renderer/backend/gles/render_pass_gles.cc

```diff
@@ -470,6 +474,51 @@ struct RenderPassData {
     }
   }
 
+  // When we have a resolve_attachment, MSAA is being used. We blit from the
+  // MSAA FBO to the resolve FBO, otherwise the resolve FBO ends up being
+  // incomplete (because it has no attachments).
+  //
+  // Note that this only works on OpenGLES 3.0+, or put another way, in older
+  // versions of OpenGLES, MSAA is not currently supported by Impeller. It's
+  // possible to work around this issue a few different ways (not yet done):
+  //
+  // Do not use the resolve_texture, and instead use the MSAA texture only.
+  // To preview what this would look like (but should not be implemented this
+  // way - would impact the HAL):
+  //
+  //    @@ render_target.cc @@ CreateOffscreenMSAA
+  //    - color0.resolve_texture = color0_resolve_tex;
+  //    + color0.resolve_texture = color0_msaa_tex;
```

Comment:
nit: I would remove this part.

Reply:
This is mostly a reminder to myself 😅 but I can put it in the issue I end up filing instead:

```
  // Do not use the resolve_texture, and instead use the MSAA texture only.
  // To preview what this would look like (but should not be implemented this
  // way - would impact the HAL):
  //
  //    @@ render_target.cc @@ CreateOffscreenMSAA
  //    - color0.resolve_texture = color0_resolve_tex;
  //    + color0.resolve_texture = color0_msaa_tex;
```